rhobs/alerting/data_plane/prometheus.application_alerts.yaml

```diff
@@ -27,3 +27,21 @@ spec:
           {{ $labels.namespace }} on cluster {{ $labels.source_cluster }} is failing to
           successfully delete at least 99% of applications over the past hour
         runbook_url: TBD
+    - alert: ApplicationCreationErrors
+      expr: |
+        (sum(increase(has_application_failed_creation_total[1h]))
+        by(namespace, pod, source_cluster) /
+        sum(increase(has_application_creation_total[1h]))
+        by(namespace, pod, source_cluster)) > 0.05
```

Comment:
Do the calls to `sum` here make any difference to the result of the individual queries? Can the combination of namespace/pod/source_cluster can potentially return more than a single time series?

Reply:
If not, then we should drop it. If it does, then we need test cases for that.

Reply:
@yftacherzog Thanks for the review. I've updated the alert accordingly. I also updated the other alert for Applications, as it used the same format.